pkg/controller/operators/openshift/helpers_test.go

```diff
@@ -219,11 +219,22 @@ func TestIncompatibleOperators(t *testing.T) {
 	}
 	for _, tt := range []struct {
 		description string
+		cv          configv1.ClusterVersion
```

Comment:
nit: the amount of noise in the test cases could be reduced if this was changed `version string`.